schemas/test/en/test_new_calculated_summary_repeating_and_static_answers.json

```
@@ -0,0 +1,726 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Calculated Summary with Dynamic Answers",
+    "theme": "default",
+    "description": "A questionnaire to demo calculated summaries which use a list of repeating answers.",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Hub",
+        "options": {}
```

Since big "Choose another section to complete" title on the hub doesn't make sense now, we should only make it visible(accessible) when the first section is complete, so:
```suggestion
        "options": {
            "required_completed_sections": [
                "section-1",
            ]
        }
```

---

Done 👍